pkgs/os-specific/linux/ati-drivers/fglrx_gpl_symbol.patch

```diff
@@ -0,0 +1,11 @@
+--- 13.1/common/lib/modules/fglrx/build_mod/firegl_public.c	2013-01-15 22:33:27.000000000 +0100
++++ 13.1/common/lib/modules/fglrx/build_mod/firegl_public.c	2014-08-18 20:05:44.327520475 +0200
+@@ -250,7 +250,7 @@
+ #endif
+ 
+ #ifdef MODULE_LICENSE
+-MODULE_LICENSE("Proprietary. (C) 2002 - ATI Technologies, Starnberg, GERMANY");
++MODULE_LICENSE("GPL\0Proprietary. (C) 2002 - ATI Technologies, Starnberg, GERMANY");
```

Comment:
These kinds of hacks don't seem nice. We're making kernel think that fglrx is licensed under GPL, which is clearly a lie. It would certainly be better to use patches like https://anonscm.debian.org/viewvc/pkg-fglrx/fglrx-driver/trunk/debian/patches/11-4.1.0-gpl-only.patch?view=markup instead (I didn't verify this particular patch is enough).